Source/ACE.Server/WorldObjects/WorldObject_Use.cs

```diff
@@ -123,10 +119,22 @@ public virtual void OnActivate(WorldObject activator)
             if (ActivationResponse.HasFlag(ActivationResponse.Use))
             {
                 if (player != null)
-                    EmoteManager.OnUse(player);
+                {
+                    //target.EmoteManager.OnActivation(player); // found a few things with Activation on them but not ActivationResponse.Emote...
+                    target.EmoteManager.OnUse(player);
+                }
+                else if (activator is Creature creature)
+                {
+                    //target.EmoteManager.OnActivation(creature); // found a few things with Activation on them but not ActivationResponse.Emote...
```

Comment:
why the duplicate logic here?
just the 2nd one is needed, would handle both players and creatures?